java/code/src/com/suse/oval/config/test/oval.config.json

```diff
@@ -0,0 +1,144 @@
+{
+  "sources": {
+    "openSUSE_LEAP": {
```

Comment:
This file I think is used just for testing. Looking at the one that will be shipped to customers the name of this OS is different. In here called `openSUSE_LEAP` and on the other file is `LEAP`. What should be the name?

Reply:
Nice catch! It was a typo. Right now, I'm thinking of using a symbolic link to the OVAL config file in `susemanager-sync-data`? @rjmateus what do you think?

Reply:
It looks like symlinks could cause issues during packaging: https://github.com/uyuni-project/uyuni/pull/563. I'll add a script to update the OVAL config file, similar to the approach taken in that PR.